src/main/java/pl/edu/mimuw/Simulation.java

```diff
@@ -0,0 +1,57 @@
+package pl.edu.mimuw;
+
+import pl.edu.mimuw.chess.Gracz;
+import pl.edu.mimuw.chess.Kolor;
+import pl.edu.mimuw.chess.Plansza;
+import pl.edu.mimuw.chess.Util;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public abstract class Simulation {
+
+  public static void szachyDoPliku(File plikDocelowy) throws IOException {
+    BufferedWriter writer = new BufferedWriter(new FileWriter(plikDocelowy));
+    Plansza.initPlansza();
```

Comment:
czemu plansza jest globalnym obiektwm 🙈 to powinna byc taka sama abstrakcja jak Gracz czy Figura, by mozna bylo stworzyc nowe plansze przez `new Plansza()` i na nich grać